openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2257,6 +2249,53 @@ The following is a non-normative example of a Credential Response containing a C
 
 <{{examples/credential_response_sd_jwt_vc.txt}}
 
+# Wallet Attestations {#walletattestation}
+
+The Wallet Attestation defined by this specification is a verifiable statement that confirms the authenticity and security properties of a Wallet to the Authorization Server. Wallets can have different architectures, such as app-based with a backend service or web-based running entirely in a browser. Wallet attestations are architecture-independent and require a backend service from the Wallet Provider. When creating a Wallet Attestation, the Wallet Provider MUST verify the Client's authenticity. Mobile application attestations provided by operating systems, like iOS's DeviceCheck or Android's Play Integrity, enable the Wallet Provider's backend to confirm communication with a legitimate instance. These mechanisms help validate the Wallet's internal integrity.
```

Comment:
I think the way the wallet attestation is created should be out of scope in this document. It creates intentionally or unintentionally restrictions wrt the concept. Just one example, a wallet attestation used by a web wallet does not need a platform attestation nor does it need to be issued by the wallet backend. It could be issued by any trusted issuer.

Reply:
Agree with Torsten here I don't think we should be making normative assertions about the Wallet Provider <-> Client relationship in this specification.